benchmarks/graphics_pipeline/GraphicsBenchmarkApp.h

```diff
@@ -414,6 +414,20 @@ class GraphicsBenchmarkApp
     std::array<grfx::ShaderModulePtr, kFullscreenQuadsTypes.size()>      mQuadsPs;
 
     BlitContext mBlit;
+    // Metrics Data
+    struct MetricsData
+    {
+        enum MetricsTypes : size_t
+        {
+            kTypeCPUSubmissionTime = 0,
+            kTypeBandwidth,
+            kCount
+        };
+
+        ppx::metrics::MetricID metrics[kCount] = {};
+    };
+    MetricsData mMetricsData;
+    uint32_t    mSkipRecordBandwidthMetricFrameCounter = 0;
```

Comment:
Could you add a short comment for this member?

Reply:
added as suggested